django_lti_tool_provider/views.py

```diff
@@ -67,41 +77,33 @@ def process_request(self, request):
             return self.process_anonymous_lti(request)
 
     @classmethod
-    def _get_lti_parameters_from_request(cls, request):
-        provider = DjangoToolProvider(settings.LTI_CLIENT_KEY, settings.LTI_CLIENT_SECRET, request.POST)
-        provider.valid_request(request)
-        return provider.to_params()
-
-    @classmethod
-    def _store_lti_parameters(cls, user, parameters):
-        """
-        Filters out OAuth parameters than stores LTI parameters into the DB, creating or updating record as needed
-        """
-        # TODO: this method got complicated - move into model/model manager
-        lti_params = {
+    def lti_param_filter(cls, parameters):
```

Comment:
@Kelketek nit: it's not only a filter, but also converter :) Would be great if name reflected this.